gcp_variant_transforms/testing/integration/preprocessor_tests/no_conflicts_from_file.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "test_name": "no-conflicts",
```

Comment:
Please use a unique test_name.

Reply:
Since this test actually passes even when we wrongly provide `input_pattern`, so it is preferred to modify the input file to actually generate some conflicts information. For instance, you can add `gs://gcp-variant-transforms-testfiles/small_tests/invalid-4.0-POS-empty.vcf` into `combine`, the expected contents should be something similar to `report_malformed_records.json`

Reply:
Done.